src/humanloop/client.py

```diff
@@ -136,22 +135,20 @@ def __init__(
             )
         instrument_provider(provider=self._tracer_provider)
         self._tracer_provider.add_span_processor(
-            HumanloopSpanProcessor(
-                client=self,
-                exporter=HumanloopSpanExporter(client=self),
-            ),
+            HumanloopSpanProcessor(exporter=HumanloopSpanExporter(client=self)),
         )
 
         if opentelemetry_tracer is None:
             self._opentelemetry_tracer = self._tracer_provider.get_tracer("humanloop.sdk")
         else:
             self._opentelemetry_tracer = opentelemetry_tracer
 
+    @contextmanager
     def prompt(
         self,
         *,
-        path: Optional[str] = None,
-        **prompt_kernel: Unpack[DecoratorPromptKernelRequestParams],  # type: ignore
+        path: str,
```

Comment:
nit for all of these: we should support passing either path or id, no?

Reply:
I'd keep it to path to reduce cognitive load